src/main/java/com/codecool/virtualpub/data/BartenderScript.java

```diff
@@ -0,0 +1,39 @@
+package com.codecool.virtualpub.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum BartenderScript {
+    // Welcome phrases
+    WELCOME(Arrays.asList(
```

Comment:
Smart enum is nice here.

I suggest using `List.of` instead of `Arrays.asList` here. `Arrays.asList` allows modifying its elements and allows `null` elements, none of which is desirable here if I understand correctly.